### ISSUE TITLE
各种卷积层 | 东方少

### ISSUE THREAD

**Issue by Wangjii** [Wangjii/GitHubPage_Comments#26]
https://wangjii.github.io/article/75a3/#%E5%8F%82%E8%80%83 

 普通卷积 1D卷积  1D卷积主要应用于时序数据，上图所示的数据有三个数据点，卷积核沿着时间维度滑动。适用于音频和文本数据。 1class torch.nn.Conv1d(in_channels, out_channels, kernel_size, stride=1, padding=0, dilation=1, groups=1, bias=True)  in_channels(int) –
